src/brogue/IO.c

```diff
@@ -2804,7 +2804,9 @@ boolean getInputTextString(char *inputText,
     maxLength = min(maxLength, COLS - x);
 
 
-    strcpy(inputText, defaultEntry);
+    if (inputText != defaultEntry) {
+        strcpy(inputText, defaultEntry);
+    }
```

Comment:
It's ideal to have separate fixes as separate commits, so they can be seen

Reply:
I'll put this in a separate PR, since it's unrelated to this one except for being discovered while working on this issue.